README.md

```diff
@@ -401,4 +401,4 @@ INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER
 RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
 ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION,
 ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE
-OF THIS SOFTWARE.
\ No newline at end of file
+OF THIS SOFTWARE.
```

Comment:
Line ending wars are best to avoid. We can leave this as is, but in the future, `git add -p` will help avoid including unrelated changes to your commit. :-)